frontend/justfile

```diff
@@ -59,7 +59,7 @@ up *flags:
 
 # Wait for all profile services to be up
 wait-up: up
-    echo "🚧 TODO"
+    just wait
```

Comment:
I think it'd be fine to move the contents of the `wait` recipe here and remove the `@wait` recipe below entirely!

Reply:
I think this is implemented this way for consistency with the API justfile which has both `@wait` and `wait-up`. I can get behind this, as this can make working in a Dockerised context locally easier in the future.